s3watcher/S3Watcher.js

```diff
@@ -0,0 +1,118 @@
+// dependencies
+var async = require('async');
+var AWS = require('aws-sdk');
+var util = require('util');
+var request = require('request');
+var fs = require('fs');
+
+// get reference to S3 client
+var s3 = new AWS.S3();
+
+exports.handler = function(event, context) {
+    var srcBucket = event.Records[0].s3.bucket.name;
+    var srcKey = decodeURIComponent(event.Records[0].s3.object.key.replace(/\+/g, " "));
+    var splitSrcKey = srcKey.split("/");
+    var simpleKey = splitSrcKey[(splitSrcKey.length - 1)];
+    var filename = '/tmp/' + simpleKey;
+
+    // TODO: Constants we want set from elsewhere!
+    var baseUrl = "nope";
```

Comment:
@gidsg do we have an established pattern for injecting secrets into Lambda functions? I was thinking perhaps pull and parse a properties file from S3.